functions/src/Controller/User.ts

```diff
@@ -35,4 +37,46 @@ export const editUserController = functions.https.onRequest(async (request, resp
 			response.status(404).send('Error 404: User document not found');
 		}
 	});
+});
+
+export const assignPositionController = functions.https.onRequest(async (request, response) => {
+	const user: User = new User(request.body[0]);
+	const committee: Committee = new Committee(request.body[1]);
+
+	const userCollection = getUserCollection();
+	const userDoc = userCollection.doc(user.id);
+
+	const committeeCollection = getCommitteeCollection();
+	const committeeDoc = committeeCollection.doc(committee.id);
+
+	let validUser = false;
+	let validCommittee = false;
+
+	await userDoc.get().then((docSnapshot) => {
```

Comment:
i don't think you need the await here because the .then() does basically the same as await.